slide/Cours_04_1_models.Rmd

```diff
@@ -27,7 +27,7 @@ penguins <- na.omit(penguins)
 ```
 
 ```{r, fig.height=4, fig.width=4, fig.align='center', echo = FALSE}
-adelie <- subset(penguins, species == "Adelie")
+adelie <- subset(penguins, species == "Adélie")
```

Comment:
Le nom de l'espèce n'a pas d'accent dans le jeu de données je pense (en anglais). Ce changement risque de casser le code.

Reply:
"Adélie" avec un accent était écrit sur d'autres lignes. J'ai juste noté la différence et modifié pour faire correspondre.